test/init.bats

```diff
@@ -35,7 +35,7 @@ load test_helper
   cd "$PYENV_TEST_DIR"
   cat > myscript.sh <<OUT
 #!/bin/sh
-eval "\$(pyenv-init -)"
+eval "\$(pyenv init -)"
```

Comment:
This is not really necessary, but only changed for consistency I assume?

Reply:
correct